mne/io/cnt/cnt.py

```diff
@@ -49,18 +50,25 @@ def read_raw_cnt(input_fname, montage, eog=(), misc=(), ecg=(), emg=(),
         EOG channels. If 'header', VEOG and HEOG channels assigned in the file
         header are used. If 'auto', channel names containing 'EOG' are used.
         Defaults to empty tuple.
-    misc : list or tuple
+    misc : list | tuple
         Names of channels or list of indices that should be designated
         MISC channels. Defaults to empty tuple.
-    ecg : list or tuple | 'auto'
+    ecg : list | tuple | 'auto'
         Names of channels or list of indices that should be designated
         ECG channels. If 'auto', the channel names containing 'ECG' are used.
         Defaults to empty tuple.
-    emg : list or tuple
+    emg : list | tuple
         Names of channels or list of indices that should be designated
         EMG channels. If 'auto', the channel names containing 'EMG' are used.
         Defaults to empty tuple.
-    preload : bool or str (default False)
+    n_bytes : int | None
```

Comment:
Would this be clearer as something like `data_format='auto' | 'int16' | 'int32'`?